### PR TITLE
Running gods over with the shuttle is canon

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
And they will play a death noise when it happens

If anyone can think of a silly message that'd be appropriate for when they get hit that'd be fun as well

### CODE REVIEW

**Comment by PKPenguin321** [comment]
[src] is flung into the deepest reaches of space at unimaginable speeds by the shuttle!

**Comment by optimumtact** [comment]
There is a faint squeaking sound as the shuttle rips the elder god in twain, it's sort of the sound of underwhelming disappointment.

**Comment by Davdichan** [comment]
All I can think of is ford prefect almost getting run over trying to say hello in hitch hikers guide

**Comment by LaharlMontogmmery** [comment]
This is a page straight out of the Old Man Henderson book.

> "Henderson here. Figured out what the nasties are weak against."

>"What's that, Mr. Henderson?"

>"Point blank annihilation."

File: code/game/gamemodes/clock_cult/clock_ratvar.dm [hunk 1]
```diff
@@ -10,6 +10,7 @@
 	poi_list += src
 
 /obj/structure/clockwork/massive/Destroy()
+	world << 'sound/magic/clockwork/anima_fragment_death.ogg'
```
Comment (Xhuis) [inline code/game/gamemodes/clock_cult/clock_ratvar.dm]: You should probably move this to `massive/ratvar` - gateways are also counted as massive structures.

**Comment by Davdichan** [comment]
[src] learned that the bigger ship gets the right of way.

**Comment by jughu** [comment]
X didnt look both way before crossing

**Comment by optimumtact** [comment]
Please finish.

**Comment by KorPhaeron** [comment]
I lost interest just merge it and someone else can add a dumb message if they want

**Comment by ChangelingRain** [comment]
You still need to do xhuis's comment.

**Pull request closed without merging**